config/kwikset/888.xml

```diff
@@ -0,0 +1,44 @@
+    <MetaDataItem id="0446" name="ZWProductPage" type="0003">https://products.z-wavealliance.org/products/2176</MetaDataItem>
+    <MetaDataItem id="0446" name="FrequencyName" type="0003">U.S. / Canada / Mexico</MetaDataItem>
+    <MetaDataItem id="0446" name="Identifier" type="0003">888</MetaDataItem>
+    <MetaDataItem name="ProductPage">https://www.kwikset.com/products/detail/888-smartcode-electronic-deadbolt-with-z-wave-technology</MetaDataItem>
+    <MetaDataItem name="Description">Extend your smart home system to your current deadbolt, enabling your existing deadbolt to wirelessly communicate with your smart home or security system
+Dramatically reduced, all-metal interior
+Works with existing select single cylinder deadbolts</MetaDataItem>
+    <MetaDataItem name="ProductSupport">http://www.kwikset.com/Customer-Support/Overview.aspx</MetaDataItem>
+    <MetaDataItem name="Name">SmartCode 888</MetaDataItem>
+    <ChangeLog>
+      <Entry author="mike240se" date="20 Jun 2020" revision="1">Initial Version</Entry>
+    </ChangeLog>
+    <MetaDataItem name="WakeupDescription">The electronic conversion kit remains asleep until an action occurs on the latch or a request is received from the RF side.
+For the RF side, it will wake up every 1 second to check if there are any requests from your smart home controller.</MetaDataItem>
+    <MetaDataItem name="ProductManual">https://products.z-wavealliance.org/ProductManual/File?folder=&amp;filename=MarketCertificationFiles/2268/Install_Guide_914C_03_09_2017.pdf</MetaDataItem>
+    <MetaDataItem name="InclusionDescription">Initiate the process to add the lock to your system at your smart home controller.
+When prompted by your smart home system to add the lock, press button “A” on the lock interior one time. The red LED will illuminate when the lock enters Add Mode.
+Please allow time for the controller to add the lock to your system.</MetaDataItem>
+    <MetaDataItem name="ExclusionDescription">Follow your smart home system’s instructions to remove the lock from the network. When prompted by the system, press button A” on the lock interior once.  The red LED will illuminate when the lock enters Remove mode. </MetaDataItem>
+    <MetaDataItem name="ResetDescription">A factory reset will delete all user codes associated with the lock and will remove itself from your smart home system.
+Please only perform a factory reset when the primary controller is missing or inoperable.
+
+To perform a factory reset, please perform the following:
+1. Remove the battery pack
+2. Press and HOLD the Program button while reinserting the battery pack.
+3. Keep holding the button for 30 seconds until the lock beeps and the status LED flashes red.
+4. Press the Program button once more.  The status LED will flash green and red several times.
+5. After a few seconds, the lock will initiate the door handing process.</MetaDataItem>
+  </MetaData>
+  <!-- Configuration Parameters -->
```

Comment:
There is no configuration values for this device?

Reply:
there are no dip switches like the other kwikset.  The lock does have 2 configurable options: auto lock on off and delay, and mute/unmute audio but from what I can tell they are only configurable through button presses on the device.  Perhaps you can tell me in #zwave on HA discord how to confirm its not possible to access these settings via zwave if you dont mind?

Reply:
command classes in mqtt: https://i.imgur.com/DPdymA2.png

Reply:
Usually the product manual (or engineering manual) will specify if there are any configuration paramaters available via Z-Wave. Anyway. I'll merge this one.

Reply:
thanks,  then there are no z-wave configurable options, i checked all the documentation.

Reply:
This doc for the Kwikset 888 shows a number of parameters selectable by z-wave.
https://www.cd-jackson.com/zwave_device_uploads/864/Install-Guide-and-Manual-888-ENG.pdf

See § 2
> 2.  Configuration Parameters The Z-Wave door lock module supports the use of the configuration command class to provide advanced configuration of the door lock over the Z-Wave network.  This section describes the configuration parameters supported by the door lock.

| Param  | Wosit |
| ------------- | ------------- |
| 33/34 | SKU |
| 35 | Int. LED en/dis |
| 36 | Auto Lock en/dis|
| 37 | Audio en/dis|
| 40 | Factory Reset |
| 47 | Motor Overload Alarm en/dis|

(I can't even get it to lock or unlock yet though.)

Reply:
Thank you, I used the doc you provided to get them added.

As far as lock/unlock, the antenna in the 888 seems quite poor, i had bad results until I added an extender near my lock.